application/classes/Ushahidi/Validator/Post/Create.php

```diff
@@ -227,6 +227,8 @@ public function checkTags(Validation $validation, $tags)
 
 	public function checkValues(Validation $validation, $attributes, $fullData)
 	{
+
+		$attributes = !empty($fullData['values']) ? $fullData['values'] : [];
```

Comment:
At one point during Uchaguzi, the $attributes value was not being passed, and validation was failing completely but I think this was actually a different issue. Related to publish from Post Detail and Timeline because the completed_stages in those circumstances was being passed as strings rather than an array of ints.

I'll change this around to reflect that instead.